src/api/rpc_api/author.rs

```diff
@@ -284,11 +347,13 @@ where
 		Signature: Encode,
 		SignedExtra: Encode,
 	{
-		self.submit_and_watch_opaque_extrinsic_until_success(
-			&extrinsic.encode().into(),
-			wait_for_finalized,
-		)
-		.await
+		let xt_status = match wait_for_finalized {
```

Comment:
no deprecated status needed in the function impl?

Reply:
I don't think so.